### PR TITLE
[Inventory] Removed fluent interfaces

### PR DESCRIPTION
**Pull request by Arminek**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | no
| License       | MIT

### CODE REVIEW

File: src/Sylius/Component/Inventory/Model/StockableInterface.php [hunk 1]
```diff
@@ -41,36 +35,26 @@ public function getInventoryName();
...
-     * Get stock on hand.
-     *
      * @return integer
      */
     public function getOnHand();
 
     /**
-     * Set stock on hand.
-     *
      * @param integer $onHand
```
Comment (stloyd) [inline src/Sylius/Component/Inventory/Model/StockableInterface.php]: `int` should be used to be consistent with Symfony.

File: src/Sylius/Component/Inventory/Model/StockableInterface.php [hunk 2]
```diff
@@ -41,36 +35,26 @@ public function getInventoryName();
     public function isInStock();
 
     /**
-     * Is stockable available on demand?
-     *
      * @return Boolean
```
Comment (stloyd) [inline src/Sylius/Component/Inventory/Model/StockableInterface.php]: `bool` should be used to be consistent with Symfony.

File: src/Sylius/Component/Inventory/Model/StockableInterface.php [hunk 3]
```diff
@@ -12,22 +12,16 @@
...
- * Stockable interface.
- *
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
 interface StockableInterface
 {
     /**
-     * Get stock keeping unit.
-     *
      * @return mixed
```
Comment (stloyd) [inline src/Sylius/Component/Inventory/Model/StockableInterface.php]: This is string IIRC.

**Comment by michalmarcinkowski** [comment]
Thank you Arek! :+1:

**Pull request merged**